upgrading/topics/keycloak/changes.adoc

```diff
@@ -36,6 +36,35 @@ please take a look at link:{developerguide_jsproviders_link}[{developerguide_jsp
 
 In the previous releases, developers were allowed to provide client credentials to the JavaScript adapter. For now on, this capability was *removed*, because client-side apps are not safe to keep secrets.
 
+==== Authentication flows changes
+
+We did some refactoring and improvements related to the authentication flows, which requires some attention during migration.
+
+OPTIONAL execution requirement removed::
+    Regarding migration, the most important change is removing the support for OPTIONAL requirement from authentication executions and
+    replacing it with the CONDITIONAL requirement, which allows more flexibility. The existing OPTIONAL authenticators configured in previous version will be replaced with the CONDITIONAL subflows, where particular subflow will have
```

Comment:
add "the" before "previous version"

Reply:
add "the" before "previous version" -> OK
add "the" before "particular subflow" -> changing to "... CONDITIONAL subflows. These subflows will have the `Condition - User Configured` condition ..."